tests/component/test_interpreter.py

```diff
@@ -36,10 +40,13 @@ def test___grpc_channel_with_errors___get_error_string___returns_failed_to_retri
     assert error_message.startswith("Failed to retrieve error description.")
 
 
-@pytest.mark.parametrize("error_code", list(_ERROR_MESSAGES))
+@pytest.mark.parametrize("error_code", list(get_error_messages()))
```

Comment:
`from nidaqmx._grpc_interpreter import _ERROR_MESSAGES`

`_ERROR_MESSAGES` is used here as well as inside the test function, so importing inside the function is making it not accessible for `parametrize` to use.

Reply:
Removed the `get_error_messages()` method.